internal_handlers.go

```diff
@@ -8,6 +8,21 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+// InternalAuthenticationGet fetches one authentication and returns it with the password exposed. Internal use only.
+func InternalAuthenticationGet(c echo.Context) error {
+	authDao, err := getAuthenticationDao(c)
+	if err != nil {
+		return err
+	}
+
+	auth, err := authDao.GetById(c.Param("uuid"))
```

Comment:
Sorry I already approved but forgot to comment, we need this functionality hidden behind that query parameter `expose_encrypted_attributes[]=password` basically do something similar to the way i did in the filtering middleware.

Heres an example request I make from the superkey worker: https://github.com/RedHatInsights/sources-superkey-worker/blob/d076299a7d68563aa563256c37c8d846412f44d3/sources/api_client.go#L58-L63

Reply:
Hm. actually maybe since it wasn't documented we can change it. Let me chew on it a bit more.

Reply:
Does this require any other change ?